test/mrb_signalthread.rb

```diff
@@ -1,66 +1,67 @@
 assert('SignalThread#trap') do
-  a = 1
+  q = Queue.new
+
   th1 = SignalThread.trap(:HUP) do
-    a = 2
+    q.push 2
```

Comment:
Sorry for the delay in confirmation 🙇 . Why does this part of IF change? Is it necessary?

Reply:
To avoid thread conflict now mruby-thread will copy whole `Proc` environments.
Only passing values with `Queue` will be supported.

Reply:
nrhd... thankyou!